src/js/messaging/containers/Compose.jsx

```diff
@@ -153,6 +153,7 @@ export class Compose extends React.Component {
           onSaveMessage={this.saveDraftIfNoAttachments}
           onSendMessage={this.sendMessage}
           onSubjectChange={this.props.setMessageField.bind(null, 'message.subject')}
+          sendingMessage={this.props.loading.sendingMessage}
```

Comment:
I decided to pass only the `sendingMessage` property instead of the whole `loading` object because that way if the boolean isn't passed it'll be falsy by default and we don't have to check for the existence of the `loading` object.